### PR TITLE
Add eheimdigital integration

### PR DESCRIPTION
**Pull request by autinerd**
## Proposed change

This adds the eheimdigital integration.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [X] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [X] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/126757
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/5935
- This PR fixes or closes issue: fixes #

## Checklist

- [X] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [X] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

## Summary by CodeRabbit

- **New Features**
	- Introduced documentation for the EHEIM Digital integration, enabling users to control EHEIM smart aquarium devices.
	- Highlighted functionalities for managing lights and daycycle mode for the EHEIM classicLEDcontrol+e device.
- **Documentation**
	- Added comprehensive setup and configuration flow guide for the EHEIM Digital integration.
	- Included essential metadata such as title, description, and supported platforms for better user understanding.

### CODE REVIEW

File: source/_integrations/eheimdigital.markdown [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+## Supported devices
+
+Currently, the following devices and features are supported:
+
+### [EHEIM classicLEDcontrol+e](https://eheim.com/en_GB/aquatics/technology/lighting-control/classicledcontrol-e/classicledcontrol-e)
+
+- Controlling the brightness of both light channels
+- Enabling the daycycle mode, which automatically controls the brightness based on the daytime as configured on the device
+
+Support for additional EHEIM Digital devices will be added in future updates.
```
Comment (c0ffeeca7) [inline source/_integrations/eheimdigital.markdown]:
```suggestion
Support for additional EHEIM Digital devices will be added in future updates.

## Entities

The EHEIM Digital integration provides the following entities:

### Lights

**Brightness**:  Allows controlling the brightness of both light channels.
**Daycycle mode**:  Automatically controls the brightness based on the daytime as configured on the device.
```
From your description, it sounds like you're adding more entities later. maybe it's worth moving this information already into a separate section.

Comment (autinerd) [reply source/_integrations/eheimdigital.markdown]: I have it now restructured.

Comment (c0ffeeca7) [reply source/_integrations/eheimdigital.markdown]: :pray: thanks!

**Review by c0ffeeca7 — DISMISSED** [review]
Thank you, @autinerd :-1:

File: source/_integrations/eheimdigital.markdown [hunk 2]
```diff
@@ -0,0 +1,45 @@
+---
+title: EHEIM Digital
+description: Instructions on how to set up EHEIM Digital with Home Assistant.
+ha_category:
+  - Light
+ha_release: 2024.11
```
Comment (joostlek) [inline source/_integrations/eheimdigital.markdown]:
```suggestion
ha_release: 2025.1
```

**Review by joostlek — APPROVED** [review]

**Pull request merged**